### ISSUE TITLE
Minimal “icon” version of Babel logo in SVG format

### ISSUE THREAD

**Issue by mathiasbynens** [babel/website#2028]
Hey y’all ♥

I often find myself wanting to mention Babel when talking about feature support. E.g.

> This feature works in Chrome X, Firefox Y, Safari Z, Node.js π, and can be transpiled using Babel.

This works well in image form, since browser logos are mostly used as icons already and the Node.js icon can easily be extracted from its logo:

![](https://user-images.githubusercontent.com/81942/57974829-d65c3a80-7994-11e9-8243-d3b347ce1ffc.png)

However, the Babel logo does not have an official icon version, and is therefore hard to combine with the above icons:

![](https://user-images.githubusercontent.com/81942/57974825-ccd2d280-7994-11e9-8cf4-d9f4d4a9f624.png)

The Babel website’s favicon uses a minimal version of the Babel logo: it’s only the letter `B` taken from the logo. This is the perfect icon to use when referring to Babel but space is constrained!

![](https://babeljs.io/img/favicon.png)

TL;DR Can this icon be made available as a separate SVG image please? (I tried editing the Babel SVG manually, but it turns out I’m pretty bad at it.)

**Comment by hzoo** [comment]
I think we can just edit the https://github.com/babel/logo/blob/master/babel.ai or psd file and then export a new svg? Could probably just add that to the repo. @existentialism said he could do it

**Comment by existentialism** [comment]
![babel-logo-minimal](https://user-images.githubusercontent.com/56288/58110630-8a3c1080-7bb5-11e9-8f16-afa391dc4223.jpg)

As SVG:

[babel-logo-minimal.svg.zip](https://github.com/babel/website/files/3203390/babel-logo-minimal.svg.zip)

**Comment by mathiasbynens** [comment]
Awesome!

**Comment by hzoo** [comment]
I do kinda want an icon logo at some point tho 😁

**Comment by xtuc** [comment]
Cool, thanks @existentialism. Could we also please commit it into our logo repo?

**Comment by sustained** [comment]
The white background makes it unusable on dark themes. :frowning:

**Comment by existentialism** [comment]
@sustained will the svg work for you?

**Comment by sustained** [comment]
Didn't realise the SVG had no background colour unlike the PNG - that works for me.

Both should probably be that way though, non-white backgrounds are common! And making transparent PNGs isn't too much effort.

**Comment by JLHwung** [comment]
A 32x32 svg version best suited for file icons / Favicon
[babel-logo-minimal-32x32.svg.zip](https://github.com/babel/website/files/3772734/babel-logo-minimal-32x32.svg.zip)

A monochormatic 32x32 svg version best suited for IDE file icons in both dark mode and light mode.
[babel-logo-monochromatic.svg.zip](https://github.com/babel/website/files/3772735/babel-logo-monochromatic.svg.zip)

**Comment by nikkipantony** [comment]
I was also looking around for a minimal Babel logo / icon and ended up creating this as I couldn't find one. I’ll leave it here in case anyone would like to use it. It's a little closer to the original logo than the above example.
![Babel_Icon](https://user-images.githubusercontent.com/3025322/87547253-bf050400-c6a2-11ea-950a-280311bc6cc8.png)
If other sizes / formats would be useful, please let me know. 😃 

SVG File: [Babel_Icon.svg.zip](https://github.com/babel/website/files/4925278/Babel_Icon.svg.zip)
PNG File: [Babel_Icon.png.zip](https://github.com/babel/website/files/4925285/Babel_Icon.png.zip)
JPG File: [Babel_Icon.jpg.zip](https://github.com/babel/website/files/4925340/Babel_Icon.jpg.zip)
AF (Affinity Designer) File: [Babel_Icon.af.zip](https://github.com/babel/website/files/4925351/Babel_Icon.af.zip)

**Comment by existentialism** [comment]
@nikkipantony thanks, the minimal example above was modified to look better at smaller sizes.

**Comment by nikkipantony** [comment]
@existentialism That makes a lot of sense if it’s being used as a favicon or other very small icon. I just thought I'd add the above version also in case it could be of any use.

**Comment by existentialism** [comment]
@nikkipantony no, no, appreciate the effort! just wanted to explain why it was different from the original "B" is all 😃
